src/V3ParseGrammar.cpp

```diff
@@ -168,6 +168,11 @@ AstVar* V3ParseGrammar::createVariable(FileLine* fileline, const string& name,
     if (GRAMMARP->m_varIO == VDirection::NONE && GRAMMARP->m_varDecl == VVarType::PORT) {
         // Just a port list with variable name (not v2k format); AstPort already created
         if (dtypep) fileline->v3warn(E_UNSUPPORTED, "Unsupported: Ranges ignored in port-lists");
+        if (arrayp) VL_DO_DANGLING(arrayp->deleteTree(), arrayp);
+        if (attrsp) {
+            // TODO: Warn that attribute is ignored here?
```

Comment:
```suggestion
            // TODO: Merge attributes across the list? Or warn attrs ignored.
```

Reply:
FWIW this is for 

```
module (clk /* verilator clocker */ <- igonred
);

input clk
endmodule
```
in t_clk_concat_2